github/repos_pages.go

```diff
@@ -62,6 +70,51 @@ func (s *RepositoriesService) EnablePages(ctx context.Context, owner, repo strin
 	}
 
 	return enable, resp, nil
+
```

Comment:
Please remove this blank link to be consistent with Go style and the rest of this repo.